trackers/javascript-tracker/rollup.config.test.js

```diff
@@ -31,14 +31,20 @@
 import { nodeResolve } from '@rollup/plugin-node-resolve';
 import ts from 'rollup-plugin-ts'; // Prefered over @rollup/plugin-typescript as it bundles .d.ts files
 import commonjs from '@rollup/plugin-commonjs';
+import alias from '@rollup/plugin-alias';
 import json from '@rollup/plugin-json';
 
 const plugins = [json(), nodeResolve({ browser: true }), commonjs(), ts({ tsconfig: './tsconfig.prod.json' })];
 
 export default [
   {
     input: './src/index.ts',
-    plugins: plugins,
+    plugins: [
+      alias({
+        entries: [{ find: '../tracker.config', replacement: '../tracker.test.config' }],
+      }),
+      ...plugins,
+    ],
```

Comment:
I imagine this might conflict with the release/3.2.0 branch now. Might want to rebase this branch on the release branch now.